post-scan-actions/azure-python-promote-or-quarantine/template.json

```
@@ -9,6 +9,20 @@
                 "description": "The name of the promote or quarantine function to create."
             }
         },
+        "functionsPackageLocation": {
+            "type": "string",
+            "defaultValue": "https://github.com/trendmicro/cloudone-filestorage-plugins/releases/download/azure-python-promote-or-quarantine-1.0.0",
```

The version should be `1.0.2`?
```suggestion
            "defaultValue": "https://github.com/trendmicro/cloudone-filestorage-plugins/releases/download/azure-python-promote-or-quarantine-1.0.2",
```

---

I think this version can be regarded as `1.0.0`, `1.0.2` is just a commit when I was testing, and it has been changed back to 1.0.0